src/components/Tabs/Tabs.stories.tsx

```diff
@@ -108,50 +126,76 @@ export default {
     tags: ['autodocs'],
     argTypes: {
         paddingX: {
-            options: Object.values(TabsPaddingX),
+            options: TABS_PADDING_X_TYPES,
             control: { type: 'select' },
-            defaultValue: TabsPaddingX.Small,
+            defaultValue: '8',
         },
         size: {
-            options: Object.values(TabSize),
+            options: TABS_SIZE_TYPES,
             control: { type: 'select' },
-            defaultValue: TabSize.Small,
+            defaultValue: 'Small',
         },
     },
 } as Meta<TabsProps>;
 
 const TabTemplate: StoryFn<TabsProps> = (args) => {
     const [activeItemId, setActiveItemId] = useState(data[0].id);
     return (
-        <Tabs {...args} activeItemId={activeItemId} onChange={(value) => setActiveItemId(value)}>
-            {data.map((item) => (
-                <TabItem id={item.id} key={item.id} label={item.label} disabled={item.disabled ?? false}>
-                    <div className="tw-p-3">{item.children}</div>
-                </TabItem>
-            ))}
-        </Tabs>
+        <div className="tw-flex tw-flex-col">
+            <Tabs {...args} activeItemId={activeItemId} onChange={(value) => setActiveItemId(value)}>
+                {data.map((item) => (
+                    <TabItem id={item.id} key={item.id} label={item.label} disabled={item.disabled ?? false}>
+                        <div className="tw-p-3">{item.children}</div>
+                    </TabItem>
+                ))}
+            </Tabs>
+        </div>
     );
 };
 export const Default = TabTemplate.bind({});
 Default.storyName = 'Label Only';
 
+const TabWithMaxHeight: StoryFn<TabsProps> = (args) => {
+    const [activeItemId, setActiveItemId] = useState(data[0].id);
+    return (
+        <div className="tw-max-h-[200px] tw-flex tw-flex-col">
```

Comment:
I think that flex class is not necessary. You could just use `tw-h-[200px]` and have the same results.

Reply:
If I remove the flex class, the scrollbar disappears. This story is just meant to showcase the scrollbar.